code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -222,6 +222,13 @@
 		post_tipped_callback = CALLBACK(src, PROC_REF(after_tip_over)), \
 		post_untipped_callback = CALLBACK(src, PROC_REF(after_righted)))
 
+	if(mapload && HAS_TRAIT(SSstation, STATION_TRAIT_MEDBOT_MANIA) && is_station_level(z))
+		skin = "advanced"
```

Comment:
Feels like this is a quite awkward way of doing it, why not replace the medibot with an advanced one?

Reply:
We don't really have any proper "advanced" subtypes for medibots, damagetype_healer and skin just get overriden for cases like Oppenheimer or the advanced medibots that can be found in space.

You are right though, I should do it in a less janky way, probably just a new subtype.

Reply:
Oh right, I remembered the other reason I did this -- roundstart medibots are mapped in and I didn't want to have them delete and replace themselves with a different subtype in Initialize, nor did I want to replace all the map spawns with spawners to avoid that problem